### DISCUSSION TITLE
The app build failed to produce artifact folder: 'build'. Could help me as I am not aware about how this things work in webdev

### DISCUSSION THREAD

**Discussion by yuvraaj2002** [microsoft/SATechnicalOnboarding#52] [Questions]
![image](https://github.com/microsoft/SATechnicalOnboarding/assets/123289647/013841eb-3302-4e37-8a84-ad80dcf9ca71)

**Comment by Nirmalkumar6112** [comment]
Make sure that you have selected the build locations as "dist" while creating Static Web App process.

Set your application information when prompted:
1.Region: pick the one closest to you
2.Project structure: select "React"
3.Location of application code: /
4.Build location: dist
